KanbanGuidelines.md

```diff
@@ -0,0 +1,49 @@
+# Kanban Guidelines
```

Comment:
I hope we can achieve that in order for the company to be more efficient. However, we have to make small incremental steps to achieve it. Just imagine the loss if this turns out not to work. So let's start with our own team, show everyone involved that it works and then promote it further. Also, if we only can use it ourselves I'd also be very happy since it's a big improvement from the current situation.